main.pm

```diff
@@ -562,6 +566,20 @@ sub load_x11tests(){
     loadtest "x11/shutdown.pm";
 }
 
+sub load_autoyast_tests(){
+#    init boot in load_boot_tests
+    loadtest("autoyast/system.pm");
+    loadtest("autoyast/console.pm");
+    loadtest("autoyast/login.pm");
+    loadtest("autoyast/repos.pm") unless get_var("SUPPORT_SERVER_GENERATOR");
+    loadtest("autoyast/autoyast_verify.pm") if get_var("AUTOYAST_VERIFY");
+    loadtest("autoyast/useradd.pm") unless get_var("INSTALLONLY");
```

Comment:
This was done to minimize modifications done to autoyast files from e.g. bugreports, but this seems not so neccessary.
Yes ok, as in the above comment, there is no problem with user to be added already into autoyast.xml.